bindings/python/examples/saveCCBToFile/saveCCBToFile.py

```diff
@@ -0,0 +1,112 @@
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+import aditofpython as tof
+import sys
+import time
+import os
+
+ccb_dir = '\\ccb_directory\\'
+ccb_prefix = 'ccb_'
+
+if __name__ == "__main__":
+
+    if len(sys.argv) < 2  or sys.argv[1] == "--help" or sys.argv[1] == "-h" :
+        print("save_ccb.py usage:")
+        print("USB / Local connection: save_ccb.py <config>")
+        print("Network connection: save_ccb.py <ip> <config>")
+        exit(1)
+
+    system = tof.System()
+
+    cameras = []
+    ip = ""
+    if len(sys.argv) == 3 :
+        ip = sys.argv[1]
+        config = sys.argv[2]
+        print (f"Looking for camera on network @ {ip}. Will use {config}.")
+        ip = "ip:" + ip
+    elif len(sys.argv) == 2 :
+        config = sys.argv[1]
+        print (f"Looking for camera on UVC. Will use {config}.")
+    else :
+        print("Too many arguments provided!")
+        exit(1)
+
+    dir_path = os.path.dirname( os.path.abspath(__file__)) + ccb_dir
+    if os.path.exists(dir_path):
+        print(f"The directory {dir_path} already exists.")
+    else:
+        # Create the directory
+        os.makedirs(dir_path)
+        print(f"The directory {dir_path} was created.")
+
+    status = system.getCameraList(cameras, ip)
+    print("system.getCameraList()", status)
+
+    camera1 = cameras[0]
+
+    status = camera1.setControl("initialization_config", config)
+    print("camera1.setControl()", status)
+
+    status = camera1.initialize()
+    print("camera1.initialize()", status)
+
+    camDetails = tof.CameraDetails()
+    status = camera1.getDetails(camDetails)
+    print("camera1.getDetails()", status)
+    print("camera1 details:", "id:", camDetails.cameraId )
+    print("connection: ", camDetails.connection)
+    print("mode: ", camDetails.mode)
+    print("mindepth: ", camDetails.minDepth)
+    print("maxdepth: ", camDetails.maxDepth)
+
+    print("Intrinsic Parameters: ")
+    # Get intrinsic parameters from camera
+    intrinsicParameters = camDetails.intrinsics
+    print("fx: ",intrinsicParameters.fx)
+    print("fy: ",intrinsicParameters.fy)
+    print("cx: ",intrinsicParameters.cx)
+    print("cy: ",intrinsicParameters.cy)
+    print("codx: ",intrinsicParameters.codx)
+    print("cody: ",intrinsicParameters.cody)
+    print("k1: ",intrinsicParameters.k1)
+    print("k2: ",intrinsicParameters.k2)
+    print("k3: ",intrinsicParameters.k3)
+    print("k4: ",intrinsicParameters.k4)
+    print("k5: ",intrinsicParameters.k5)
+    print("k6: ",intrinsicParameters.k6)
+    print("p2: ",intrinsicParameters.p2)
+    print("p1: ",intrinsicParameters.p1)
+
+    timestamp =  time.strftime('%y%m%d%H%M')
+    status = camera1.setControl("saveModuleCCB",dir_path + ccb_prefix +  timestamp + '.ccb')
```

Comment:
This function is no longer called with setControl, you need to use this function:
https://github.com/analogdevicesinc/ToF/blob/master/sdk/include/aditof/camera.h#L200